src/components/item/item-options.ts

```diff
@@ -39,14 +40,23 @@ export class ItemOptions {
    */
   @Output() ionSwipe: EventEmitter<ItemSliding> = new EventEmitter<ItemSliding>();
 
-  constructor(private _elementRef: ElementRef, private _renderer: Renderer) {}
+  constructor(private _elementRef: ElementRef, private _renderer: Renderer, private _plt: Platform) {}
 
   /**
    * @hidden
    */
   getSides(): number {
-    if (isPresent(this.side) && this.side === 'left') {
-      return ITEM_SIDE_FLAG_LEFT;
+    if (isPresent(this.side)) {
+      switch(this.side) {
+        case 'left':
+          return ITEM_SIDE_FLAG_LEFT;
+        case 'right':
+          return ITEM_SIDE_FLAG_RIGHT;
+        case 'start':
+          return _plt.isRTL() ? ITEM_SIDE_FLAG_RIGHT : ITEM_SIDE_FLAG_LEFT;
+        case 'end':
+          return _plt.isRTL() ? ITEM_SIDE_FLAG_LEFT : ITEM_SIDE_FLAG_RIGHT;
+      }
     }
     return ITEM_SIDE_FLAG_RIGHT;
```

Comment:
Should this be changed to:

```
return this._plt.isRTL() ? ITEM_SIDE_FLAG_LEFT : ITEM_SIDE_FLAG_RIGHT;
```

So that it automatically switches to the other side when RTL? Or is this undesired behavior?